runelite-client/src/main/java/net/runelite/client/plugins/interfacestyles/InterfaceStylesPlugin.java

```diff
@@ -310,4 +339,98 @@ private void removeGameframe()
+
+		return membersOnlyWidgets;
+	}
+
+	private void shuffleSkills(boolean hideMembers)
+	{
+		if (hideMembers && client.getWidget(WidgetInfo.SKILL_TOTAL) != null)
+		{
+			client.getWidget(WidgetInfo.SKILL_TOTAL).setRelativeX(WOODCUTTING_X);
+			client.getWidget(WidgetInfo.SKILL_TOTAL).setRelativeY(WOODCUTTING_Y);
+			client.getWidget(WidgetInfo.SKILL_PRAYER).setRelativeX(client.getWidget(WidgetInfo.SKILL_AGILITY).getRelativeX());
+			client.getWidget(WidgetInfo.SKILL_PRAYER).setRelativeY(client.getWidget(WidgetInfo.SKILL_AGILITY).getRelativeY());
+			client.getWidget(WidgetInfo.SKILL_MAGIC).setRelativeX(PRAYER_X);
+			client.getWidget(WidgetInfo.SKILL_MAGIC).setRelativeY(PRAYER_Y);
+			client.getWidget(WidgetInfo.SKILL_CRAFTING).setRelativeX(client.getWidget(WidgetInfo.SKILL_HERBLORE).getRelativeX());
+			client.getWidget(WidgetInfo.SKILL_CRAFTING).setRelativeY(client.getWidget(WidgetInfo.SKILL_HERBLORE).getRelativeY());
+			client.getWidget(WidgetInfo.SKILL_RUNECRAFTING).setRelativeX(client.getWidget(WidgetInfo.SKILL_THIEVING).getRelativeX());
+			client.getWidget(WidgetInfo.SKILL_RUNECRAFTING).setRelativeY(client.getWidget(WidgetInfo.SKILL_THIEVING).getRelativeY());
+			client.getWidget(WidgetInfo.SKILL_WOODCUTTING).setRelativeX(CRAFTING_X);
+			client.getWidget(WidgetInfo.SKILL_WOODCUTTING).setRelativeY(CRAFTING_Y);
+		}
+		else if (client.getWidget(WidgetInfo.SKILL_TOTAL) != null)
+		{
+			client.getWidget(WidgetInfo.SKILL_TOTAL).setRelativeX(TOTAL_X);
+			client.getWidget(WidgetInfo.SKILL_TOTAL).setRelativeY(TOTAL_Y);
+			client.getWidget(WidgetInfo.SKILL_PRAYER).setRelativeX(PRAYER_X);
+			client.getWidget(WidgetInfo.SKILL_PRAYER).setRelativeY(PRAYER_Y);
+			client.getWidget(WidgetInfo.SKILL_MAGIC).setRelativeX(MAGIC_X);
+			client.getWidget(WidgetInfo.SKILL_MAGIC).setRelativeY(MAGIC_Y);
+			client.getWidget(WidgetInfo.SKILL_CRAFTING).setRelativeX(CRAFTING_X);
+			client.getWidget(WidgetInfo.SKILL_CRAFTING).setRelativeY(CRAFTING_Y);
+			client.getWidget(WidgetInfo.SKILL_RUNECRAFTING).setRelativeX(RUNECRAFTING_X);
+			client.getWidget(WidgetInfo.SKILL_RUNECRAFTING).setRelativeY(RUNECRAFTING_Y);
+			client.getWidget(WidgetInfo.SKILL_WOODCUTTING).setRelativeX(WOODCUTTING_X);
+			client.getWidget(WidgetInfo.SKILL_WOODCUTTING).setRelativeY(WOODCUTTING_Y);
+		}
+	}
+
+	private void showMemberOnlySkills()
+	{
+		for (Widget widget : getMemberSkills())
+		{
+			if (widget != null)
+			{
+				widget.setHidden(config.hideMembersSkills());
+			}
+		}
+		shuffleSkills(config.hideMembersSkills());
+		updateTotalLevels();
+	}
+
+	private void updateTotalLevels()
+	{
+		if (client.getWidget(WidgetInfo.SKILL_TOTAL_TEXT) == null)
+		{
+			return;
+		}
+
+		int totalLevel = 0;
+
+		for (int skill = 1; skill <= 23; skill++)
+		{
+			Widget currentSkill = client.getWidget(WidgetID.STATS_GROUP_ID, skill);
+
+			if (!currentSkill.isHidden() && currentSkill.getDynamicChildren().length >= 4)
+			{
+				// [4] is the denominator of the skill fraction widget
```

Comment:
Make this a constant so it's not a magic number anymore.

Reply:
No longer required following refactor